facia-json/src/main/scala/com/gu/facia/client/models/Config.scala

```diff
@@ -59,7 +60,8 @@ object Metadata extends StrictLogging {
     "EventAltPalette" -> EventAltPalette,
     "Podcast" -> Podcast,
     "LongRunningAltPalette" -> LongRunningAltPalette,
-    "SombreAltPalette" -> SombreAltPalette
+    "SombreAltPalette" -> SombreAltPalette,
+    "SpecialReportAlt" -> SpecialReportAlt
```

Comment:
Can we please go with `SpecialReportAltPalette` for consistency?

Reply:
@ioannakok that sounds sensible will do 👍

Reply:
Perfect, thank you very much!